kedro-datasets/kedro_datasets/datasets/README.md

```diff
@@ -0,0 +1,18 @@
+# Datasets
```

Comment:
I would just add this to the top-level readme instead of having two separate ones.

Reply:
Additionally I would keep the language consistent and either say data "connectors" or data "descriptions" but not both. My preference goes to data connectors, because that's used in the main Kedro docs as well.